src/app/view/project/project-dialog/project-form/project-form.component.ts

```diff
@@ -44,17 +44,25 @@ export class ProjectFormComponent implements OnInit {
   }
 }
 
-const userIdsValidator = (control: UntypedFormControl) => {
+export const userIdsAsEmailValidator = (control: UntypedFormControl) => {
   if (!control) {
     return null;
   }
   const userIds: string[] = control.value;
-  const invalidIds = userIds.filter((id) => !id.match(/^(u|ue|e)\d+$/));
-  if (invalidIds.length === 0) {
+  // email adresse validator: regex to match emails using the expression
+  const invalidEmailPattern = userIds.filter((id) => {
+    const retVal =
+      id.match(/^([a-z0-9_\\.-]+)@([\da-z\\.-]+)\.([a-z\\.]{2,6})|(u|ue|e)\d+$/);
```

Comment:
Same as https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-backend/pull/16/files#r1698074806: this is too restrictive.